doc/crm.8.adoc

```diff
@@ -2104,6 +2104,71 @@ Example:
 utilization xen1 set memory 4096
 ...............
 
+[[cmdhelp.sbd,SBD management]]
+=== `sbd` - SBD management
+
+This level is for managing the SBD (STONITH Block Device) daemon.
+
+[[cmdhelp.sbd.configure,configure SBD]]
+==== `configure`
+
+Configure the SBD daemon for both disk-based and disk-less mode.
+
+Main functionailities include:
+- Show configured disk metadata
+- Show contents of /etc/sysconfig/sbd
+- Show SBD related cluster properties
+- Newly setup SBD configuration on a running cluster
+- Update the existing parameters
+- Add more devices to the existing disk-based SBD configuration
+
+For more details on SBD and related parameters, please see man sbd(8).
+
+Usage:
+...............
+# For disk-based SBD
+crm sbd configure show [disk_metadata|sysconfig|property]
+crm sbd configure [device=<dev>]... [watchdog-device=<dev>] [watchdog-timeout=<integer>] [allocate-timeout=<integer>] [loop-timeout=<integer>] [msgwait-timeout=<integer>]
```

Comment:
Could have improvement for the disk-based sbd scenarios, there is no point to
" Update SBD_WATCHDOG_TIMEOUT in /etc/sysconfig/sbd "

```
tws-corosync3-2:~ # crm sbd configure device=/dev/vdb28 watchdog-timeout=2
INFO: Initializing SBD device /dev/vdb28
INFO: Update SBD_WATCHDOG_TIMEOUT in /etc/sysconfig/sbd: 2
```

Reply:
Here I want to log any value changed in /etc/sysconfig/sbd
"Update SBD_WATCHDOG_TIMEOUT in /etc/sysconfig/sbd" was corresponding to the given args `watchdog-timeout=2`
Does it make no sense?

Reply:
> For the disk-based sbd:
> 
> 1. I do suggest to delete such information which is misleading a little.
>    > INFO: Update SBD_WATCHDOG_TIMEOUT in /etc/sysconfig/sbd: 2
> 2. and, I event suggest to comment it out in /etc/sysconfig/sbd, eg.
>    #SBD_WATCHDOG_TIMEOUT=5

That means SBD_WATCHDOG_TIMEOUT in sysconfig/sbd is useless?

Reply:
And I am confused by `crm sbd configure watchdog-timeout=...`. There are 3 similar items: `Timeout (watchdog) :`, `SBD_WATCHDOG_TIMEOUT=` and `stonith-watchdog-timeout=`. Which ones are expected be modified by this command?

If `Timeout (watchdog) :` and `SBD_WATCHDOG_TIMEOUT=` controls the same thing, and only one of them is effective, we should show only the effective one in `crm sbd configure show`, or indicate which one is effective in some way.

More generally, the information given by `crm sbd configure show` is too low-level. It just dumps a lot of raw configuration. It is not symmetrical with command `crm sbd configure <name>=<value>`. As a pair of _getter_ and _setter_ command, I expected `crm sdb configure show` to give me back what I specified in `crm configure <name>=<value>`.